sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSourceResolution.scala

```diff
@@ -85,6 +85,28 @@ case class DataSourceResolution(
           .asTableCatalog
       convertCTAS(catalog, identifier, create)
 
+    case DescribeColumnStatement(
+        AsTableIdentifier(tableName), colName, isExtended) =>
+      DescribeColumnCommand(tableName, colName, isExtended)
+
+    case DescribeColumnStatement(
+        CatalogObjectIdentifier(Some(catalog), ident), colName, isExtended) =>
+      throw new AnalysisException("Describing columns is not supported for v2 tables.")
```

Comment:
Should this be supported eventually, or is it redundant if DESCRIBE TABLE is available?

Reply:
Think we need to support it eventually if only to keep parity with V1 tables.